bower.json

```diff
@@ -2,7 +2,7 @@
   "name": "ember-twiddle",
   "version": "0.8.0",
   "dependencies": {
-    "ember": "2.4.4",
+    "ember": "release",
```

Comment:
Waiting for 2.5.1 to be released. 

-------- Original message --------
From: Joost de Vries <notifications@github.com> 
Date: 04/19/2016  4:20 PM  (GMT-05:00) 
To: ember-cli/ember-twiddle <ember-twiddle@noreply.github.com> 
Cc: Gaurav Munjal <gaurav9576@yahoo.com>, Author <author@noreply.github.com> 
Subject: Re: [ember-cli/ember-twiddle] Upgrade to Ember CLI 2.6 beta 1 (#405) 

In bower.json:
> @@ -2,7 +2,7 @@
>    "name": "ember-twiddle",
>    "version": "0.8.0",
>    "dependencies": {
> -    "ember": "2.4.4",
> +    "ember": "release",

why don't we require a specific version here?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub